test/puppetlabs/trapperkeeper/services/authorization/authorization_core_test.clj

```diff
@@ -105,116 +105,153 @@
+          (validate-auth-config-rule! (assoc testrule :deny ["one.anem" 23]))))
 
     (is (thrown-with-msg?
-         IllegalArgumentException
-         #".* Dangling meta character '\*' near index 0[.\s]*"
-         (validate-auth-config-rule!
-          (-> testrule
-              (assoc-in [:match-request :type] "regex")
-              (assoc-in [:match-request :path] "*.")))))
+          IllegalArgumentException
+          #".* Dangling meta character '\*' near index 0[.\s]*"
+          (validate-auth-config-rule!
+            (-> testrule
+                (assoc-in [:match-request :type] "regex")
+                (assoc-in [:match-request :path] "*.")))))
 
     (is (thrown-with-msg?
-         IllegalArgumentException
-         #"Rule query-params must be a map."
-         (validate-auth-config-rule!
-          (assoc-in testrule [:match-request :query-params] []))))
+          IllegalArgumentException
+          #"Rule query-params must be a map."
+          (validate-auth-config-rule!
+            (assoc-in testrule [:match-request :query-params] []))))
 
     (is (thrown-with-msg?
-         IllegalArgumentException
-         #".* It should be a string."
-         (validate-auth-config-rule!
-          (assoc-in testrule [:match-request :query-params]
-                    {0 []}))))
+          IllegalArgumentException
+          #".* It should be a string."
+          (validate-auth-config-rule!
+            (assoc-in testrule [:match-request :query-params]
+                      {0 []}))))
 
     (is (thrown-with-msg?
-         IllegalArgumentException
-         #".* It should be a string or list of strings."
-         (validate-auth-config-rule!
-          (assoc-in testrule [:match-request :query-params]
-                    {:env :notastringorlist}))))
+          IllegalArgumentException
+          #".* It should be a string or list of strings."
+          (validate-auth-config-rule!
+            (assoc-in testrule [:match-request :query-params]
+                      {:env :notastringorlist}))))
 
     (is (thrown-with-msg?
-         IllegalArgumentException
-         #".* contains one or more values that are not strings."
-         (validate-auth-config-rule!
-          (assoc-in testrule [:match-request :query-params]
-                    {:env [:notastring]}))))
+          IllegalArgumentException
+          #".* contains one or more values that are not strings."
+          (validate-auth-config-rule!
+            (assoc-in testrule [:match-request :query-params]
+                      {:env [:notastring]}))))
 
     (is (thrown-with-msg?
-         IllegalArgumentException
-         #".* 'delete', 'get', 'head', 'post', 'put'"
-         (validate-auth-config-rule!
-          (assoc-in testrule [:match-request :method] "gross"))))
+          IllegalArgumentException
+          #".* 'delete', 'get', 'head', 'post', 'put'"
+          (validate-auth-config-rule!
+            (assoc-in testrule [:match-request :method] "gross"))))
```

Comment:
I can go back and fix this is it will end the world... otherwise I'm ok with this if ya'll are.